include/xlnt/worksheet/page_setup.hpp

```diff
@@ -162,6 +164,8 @@ struct XLNT_API page_setup
 
     /// <summary>
     /// Returns the factor by which the page should be scaled during printing.
+    /// Assumes that this page setup has a scale (please call has_scale() to check).
```

Comment:
This could also return 100 if not set, as it is the default scale value.

Reply:
Looking at some real xlsx file, the OOXML specs seems to be correct. No decimals are allowed. The unit test `worksheet_test_suite::test_page_setup` seems wrong to me.